Jenkinsfile

```
@@ -181,7 +181,7 @@ spec:
             // If this branch name exists in integration-tests, use that
             // Otherwise, default to integration-tests/develop
             dir('Palisade-integration-tests') {
-                git url: 'https://github.com/gchq/Palisade-integration-tests.git'
+                git branch: 'develop', url: 'https://github.com/gchq/Palisade-integration-tests.git'
                 sh "git checkout ${GIT_BRANCH_NAME} || git checkout develop"
```

It'll first checkout develop, go to checkout GIT_BRANCH_NAME but if none exists, it'll still be on develop and then will still build, therefore you can remove the or
```suggestion
                sh "git checkout ${GIT_BRANCH_NAME}"
```